lib/ansible/modules/apt.py

```diff
@@ -557,7 +557,8 @@ def expand_pkgspec_from_fnmatches(m, pkgspec, cache):
                 matches = fnmatch.filter(pkg_name_cache, pkgname_pattern)
 
                 if not matches:
-                    m.fail_json(msg="No package(s) matching '%s' available" % str(pkgname_pattern))
+                    if not to_rmv:
+                        m.fail_json(msg="No package(s) matching '%s' available" % str(pkgname_pattern))
```

Comment:
issue a warning instead of a fail on the else: in case the user misspelled the package

Reply:
@bcoca Thank you for the review! It's done.